### PR TITLE
Thank you for contributing to `cats-parse`!

### PR DESCRIPTION
**Pull request by regadas**
This is a kind reminder to run `sbt +prePR` and commit the changed files, if any, before submitting.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,6 +4,8 @@ import Dependencies._
 addCommandAlias("fmt", "; scalafmtAll; scalafmtSbt")
 addCommandAlias("fmtCheck", "; scalafmtCheckAll; scalafmtSbtCheck")
 
+addCommandAlias("prePR", "fmt")
```
Comment (johnynek) [inline build.sbt:7]: Should we add test into that list? And maybe bench/compile?

**Review by johnynek — COMMENTED** [review]
Thanks for taking this!

File: build.sbt (hunk 1 above)
Comment (regadas) [reply build.sbt:7]: yup let's do it!

**Review by johnynek — APPROVED** [review]
This is awesome, thanks.

**Pull request merged**